contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -702,6 +711,11 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
     );
     receipt.premiumInJuels = SafeCast.toUint96(premiumHexaicosaUSD / paymentParams.linkUSD);
 
+    receipt.billingToken = paymentParams.billingToken;
+    receipt.linkUSD = SafeCast.toUint96(paymentParams.linkUSD);
+    receipt.nativeUSD = SafeCast.toUint96(paymentParams.nativeUSD);
+    receipt.billingUSD = SafeCast.toUint96(paymentParams.billingTokenParams.priceUSD);
```

Comment:
is `uint96` definitely big enough to hold these? based on the struct, we could make these `uint128` with no additional storage requirements

Reply:
I think uint96 should be enough:
The max number we can hold with uint96 is approximately 10^27. 
For the 3 rates, we require the the Feed to be 8 decimals. So the ratio can be as big as 10^19 for Native, LINK and Billing Token.

Let me know if the above makes sense. Not a strong opinion, happy to update~

Reply:
SGTM 👍